lib/Extension/ExpressionExtension.php

```diff
@@ -195,7 +198,7 @@ public function load(Container $container): void
         $container->register(HighlightingNodePrinter::class, function (Container $container) {
             return new HighlightingNodePrinter(
                 $container->get(NodePrinters::class),
-                new Standard8ColorMap()
+                new SolarisedColorMap()
```

Comment:
the theme should be selected based on the Symfony Console version (I think 256 colors only supported in 5)